### DISCUSSION TITLE
enable or interface_enabled

### DISCUSSION THREAD

**Discussion by srem1** [markqvist/Reticulum#212] [Help & Questions]
in config file i find https://unsigned.io/private-messaging-over-lora/ interface in my copy program only 'eneble'

whitch is ok?

[interfaces]
  [[Default Interface]]
    type = AutoInterface
    interface_enabled = True

**Comment by markqvist** [comment]
Please don't use the Issue tracker for generic questions and help requests. Moving this to the discussions.

I'm afraid I don't understand what you are asking here. Can you be more specific?

**Reply by srem1** [reply]
this issue is about DOCUMENTATIONS ;)

**Reply by Swissbandit** [reply]
interface_enabled was used before, but enabled = True is also working now

**Reply by srem1** [reply]
I sugest change the descriptions for more readable and less to get wrong.
